code/modules/projectiles/projectile.dm

```diff
@@ -999,14 +1225,14 @@
  */
 /proc/calculate_projectile_angle_and_pixel_offsets(atom/source, atom/target, modifiers)
 	var/angle = 0
-	var/p_x = LAZYACCESS(modifiers, ICON_X) ? text2num(LAZYACCESS(modifiers, ICON_X)) : ICON_SIZE_X / 2 // ICON_(X|Y) are measured from the bottom left corner of the icon.
-	var/p_y = LAZYACCESS(modifiers, ICON_Y) ? text2num(LAZYACCESS(modifiers, ICON_Y)) : ICON_SIZE_Y / 2 // This centers the target if modifiers aren't passed.
+	var/p_x = LAZYACCESS(modifiers, ICON_X) ? text2num(LAZYACCESS(modifiers, ICON_X)) : ICON_SIZE_X * 0.5 // ICON_(X|Y) are measured from the bottom left corner of the icon.
```

Comment:
just like, for the record, the standards page is stupid. if you mean to divide by 2 and that is CLEARER then DO THAT, it's a brainworms optimization not a style recommendation. goes for the whole pr

Reply:
indeed